tcf_website/models/models.py

```diff
@@ -286,6 +286,15 @@ def code(self):
         """Returns the courses code string."""
         return f"{self.subdepartment.mnemonic} {self.number}"
 
+    def eval_link(self):
+        """Returns link to student eval page for that class"""
+        link = "https://evals.itc.virginia.edu/course-selectionguide/pages/SGMain.jsp?cmp="
+        link += self.subdepartment.mnemonic
+        link += ","
+        link += str(self.number)
+        return link
+        # return "test"
```

Comment:
Remove comment!

Reply:
I thought I removed it in one of the commits...must've accidentally not added it to the commit, will fix ASAP

Reply:
Turns out I didn't hit the save button in VSCode after I deleted the line, should be fixed now